### PR TITLE
Release notes for 1.7.7

### PR DESCRIPTION
**Pull request by JimmyCYJ**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.7.x/announcing-1.7.7/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Changes
+
+- **Fixed** an issue of using explicitly empty revision flag on install.
+  ([Issue #26940](https://github.com/istio/istio/issues/26940))
+- **Fixed** the CA’s certificate signature algorithm to be the default algorithm corresponding to the CA’s signing key type.
+  ([Issue #27238](https://github.com/istio/istio/issues/27238))
+- **Fixed** an issue showing unnecessary warnings when downgrading to a lower version of Istio.
+  ([Issue #29183](https://github.com/istio/istio/issues/29183))
+- **Fixed** an issue that newer control plane installations are removing permissions for `rbac.istio.io` from istiod, causing older control planes relying on that CRD group to hang on restart.
```
Comment (brian-avery) [inline content/en/news/releases/1.7.x/announcing-1.7.7/index.md:24]:
```suggestion
- **Fixed** an issue causing older control planes relying on the `rbac.istio.io` CRD group to hang on restart due to the fact that newer control plane installations remove those permissions from istiod, .
```

**Review by brian-avery — APPROVED** [review]

File: content/en/news/releases/1.7.x/announcing-1.7.7/index.md (hunk 1 above)
Comment (ericvn) [reply content/en/news/releases/1.7.x/announcing-1.7.7/index.md:24]: If you just take Brians change (which I like), you'll need to remove the last `,`.

**Review by jwendell — COMMENTED** [review]
Missing the wasm memory leak fix?

File: content/en/news/releases/1.7.x/announcing-1.7.7/index.md (hunk 1 above)
Comment (JimmyCYJ) [reply content/en/news/releases/1.7.x/announcing-1.7.7/index.md]: Thanks for catching that. Done.

**Comment by JimmyCYJ** [comment]
> Missing the wasm memory leak fix?

Thanks. Added now, could you please take a look?

File: content/en/news/releases/1.7.x/announcing-1.7.7/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- **Fixed** an issue of using explicitly empty revision flag on install.
+  ([Issue #26940](https://github.com/istio/istio/issues/26940))
+- **Fixed** the CA’s certificate signature algorithm to be the default algorithm corresponding to the CA’s signing key type.
+  ([Issue #27238](https://github.com/istio/istio/issues/27238))
+- **Fixed** an issue showing unnecessary warnings when downgrading to a lower version of Istio.
+  ([Issue #29183](https://github.com/istio/istio/issues/29183))
+- **Fixed** an issue causing older control planes relying on the `rbac.istio.io` CRD group to hang on restart due to the fact that newer control plane installations remove those permissions from istiod.
+  ([Issue #29364](https://github.com/istio/istio/issues/29364))
+- **Fixed** a memory leak from WASM `NullPlugin` `onNetworkNewConnection`.
```
Comment (ericvn) [inline content/en/news/releases/1.7.x/announcing-1.7.7/index.md:26]: memory leak `in` instead of `from`?

Comment (JimmyCYJ) [reply content/en/news/releases/1.7.x/announcing-1.7.7/index.md]: Thanks, done

**Review by jwendell — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Pull request merged**